src/libos/src/vm/process_vm.rs

```diff
@@ -42,6 +45,21 @@ impl<'a, 'b> ProcessVMBuilder<'a, 'b> {
         self
     }
 
+    // Generate a random address within [0, range]
+    // Note: This function doesn't gurantee alignment
+    fn get_randomize_offset(range: usize) -> usize {
+        if cfg!(debug_assertions) {
+            return 0;
```

Comment:
I think the better default implementation is "return rang;". Because if we use rang as the default, and the application works, then we could know it should work with any random value.

Reply:
Done. Thank you for your review.